### PR TITLE
Edits to supplementary data/figures

### PR DESCRIPTION
**Pull request by jjc2718**
Edits to supplement to correspond with revisions/reviewer response.

### CODE REVIEW

File: content/03.methods.md [hunk 1]
```diff
@@ -49,6 +49,8 @@ For each target gene, in order to ensure that the training dataset was reasonabl
 After applying these filters, the number of valid cancer types remaining for each gene varied based on the set of samples used: more data types resulted in fewer shared samples, and fewer samples generally meant fewer valid cancer types.
 In some cases, this resulted in genes with no valid cancer types, which we dropped from the analysis.
 Out of the 511 genes from the "merged" cancer gene set described in the previous section, for the analysis using {expression, mutation} data we retained 268 target genes, for the {expression, mutation, 27k methylation, 450k methylation} analysis we retained 272 genes, and for the analysis using all data types we retained 217 genes.
+We also tried an alternate approach of including/excluding entire genes using our filters rather than individual cancer types: that is, we trained a classifier for each gene where all cancer types combined had at least 5% mutated samples and at least 100 total mutated samples.
+Results are shown in Supplementary Figure {@fig:all_volcano_filter}.
```
Comment (cgreene) [inline content/03.methods.md:53]: We might get yelled at for this. A reference to supplementary results should really go within results somewhere. Probably needs a 1-liner somewhere in the results.

**Review by cgreene — COMMENTED** [review]
One small comment - I think the reference to the results with one brief interpretation sentence needs to go to the results.

File: content/04.results.md [hunk 2]
```diff
@@ -30,6 +30,7 @@ To evaluate whether using known cancer-related genes tends to improve prediction
 We compared performance on this curated gene set with performance on an equal number of random genes (n=268) and an equal number of the most mutated genes in TCGA (n=268).
 For all gene sets, we used only the set of TCGA samples for which both gene expression and somatic mutation data exists, resulting in a total of 9,074 samples across all 33 cancer types.
 This set of samples was further filtered for each target gene to cancer types containing at least 15 mutated samples and at least 5% of samples mutated for that cancer type.
+As an alternate approach, we tried including/excluding entire genes using similar filters; results are shown in Supplementary Figure {@fig:all_volcano_filter}.
```
Comment (cgreene) [inline content/04.results.md:33]:
```suggestion
As an alternate approach, we tried including/excluding entire genes using similar filters, and the results were consistent across filtering strategies (Supplementary Figure {@fig:all_volcano_filter}).
```

Comment (cgreene) [reply content/04.results.md:33]: Is this right? If so 🚀

**Review by cgreene — APPROVED** [review]
I think one small suggestion to make a judgment call

File: content/04.results.md (hunk 2 above)
Comment (jjc2718) [reply content/04.results.md:33]: I would say this is accurate!

**Pull request merged**